db/records/operations/insert.py

```diff
@@ -1,4 +1,5 @@
 import json
+import pandas as pd
```

Comment:
Why are we renaming the import here?

Reply:
I'm used to seeing `pandas` renamed to `pd`, but most people's readability would probably be improved without this shorthand.

Reply:
Agreed re: readability.

Reply:
Updated it now. Thanks!